src/applications/health-care-supply-reordering/components/FooterInfo.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import { DLC_PHONE } from '../constants';
 
 const FooterInfo = () => (
   <section className="need-help-footer row vads-u-padding-x--1p5">
     For benefit-related questions, or if the form isn’t working right, please
-    call VA Benefits and Services at{' '}
-    <va-telephone contact={CONTACTS.VA_BENEFITS} />. If you have hearing loss,
-    call <va-telephone contact={CONTACTS['711']} tty />.
+    call VA Benefits and Services at <va-telephone contact={DLC_PHONE} />. If
```

Comment:
> VA Benefits and Services

Do we still call it that if the number goes directly to the DLC?

Reply:
Thanks. I was laser-focused on changing the phone number and completely ignored the content